server/src/main/java/org/elasticsearch/index/mapper/MappedFieldType.java

```diff
@@ -83,6 +87,12 @@ protected MappedFieldType(MappedFieldType ref) {
     }
 
     public MappedFieldType() {
+        this.init();
+    }
+
+    // intended to only be overridden by AliasFieldMapper
+    void init()
```

Comment:
Introduced this method because AliasFieldType doesnt want to do the setTokenised and other defaults, so this is my way (via polymorphism) of stopping these defaults.